src/clr-core/test/utils.ts

```diff
@@ -40,6 +42,10 @@ export function getComponentSlotContent(component: HTMLElement): { [name: string
   );
 }
 
+export const getComponentSelector = curryN(2, (tagName: string, additional = ''): string => {
+  return tagName + additional;
+});
```

Comment:
I am not sure if this is needed based on comment above.

Reply:
Yup. Will remove.

Reply:
✅